level1/alb.tf

```diff
@@ -0,0 +1,46 @@
+#Configure application load balancer
+resource "aws_lb" "alb" {
+  name               = "alb"
+  internal           = false
+  load_balancer_type = "application"
+  security_groups    = [aws_security_group.alb_sg.id]
+  #count              = length(local.public_cidr)
+  #subnets            = aws_subnet.public.*.id
+  subnets            = [for subnet in aws_subnet.public : subnet.id]
+
+  tags = {
+    Name = "custom-alb"
+  }
+}
+
+#configure security group
+resource "aws_security_group" "alb_sg" {
+  name   = "alb_sg"
+  vpc_id = aws_vpc.example.id
+
+  ingress {
+    from_port   = 80
+    to_port     = 80
+    protocol    = "tcp"
+    cidr_blocks = ["0.0.0.0/0"]
```

Comment:
It's better to close it down to the ISP range as well

Reply:
I dont understand this one..what is ISP range?